Pratiksha/html/PratikshaA5/A5.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="A5.css">
+</head>
+<body >
+    <div class="whole ">  
+        <div >
+            <h1>
+                Pratiksha P Ghule
+            </h1>
+        </div>
+        <div class="bg">
+            <div class="about"> About me</div>
+            <p class="orange">
+                Pratiksha P Ghule is a director of brand marketing, with experience managing global teams and multi-million-dollar campaigns. Her background in brand strategy, visual design, and account management inform her mindful but competitive approach. Madison is fueled by her passion for understanding the nuances of cross-cultural advertising. She considers herself a ‘forever student,’ eager to both build on her academic foundations in psychology and sociology and stay in tune with the latest digital marketing strategies through continued coursework.</p>
```

Comment:
p tag is not propar closed last line in first p tag

Reply:
Done